index.js

```diff
@@ -28,6 +28,18 @@ function serviceWorker (name, opts) {
 
     emitter.on(state.events.DOMCONTENTLOADED, function () {
       opts = mutate({ scope: '/' }, opts)
+      // electron support
+      if (opts.electron) {
+        var path = require('path')
+        var url = require('url')
+        opts.scope = path.join(__dirname, '/')
```

Comment:
Maybe we should still use the user defined scope, but with full path, because is a common patter to have electron renderer process in a custom folder

Reply:
yeah, that might be good